docs/modules/ROOT/pages/explanation/decisions/no_rbac_creation.adoc

```diff
@@ -0,0 +1,26 @@
+= Omit creation of default RBAC rules on select organization namespaces
+
+== Problem
+
+The "Managed by VSHN" appcat services provision the instances in a separate namespace that has the same organization label as the namespace where the claim originates from. By default, this will give the organization users full access to those managed namespaces and would allow the users to manipulate the managed appcat services in an undesired way.
+To avoid that, the default RBACs should not be created for those namespaces.
```

Comment:
I recommend not binding that problem to a particular product. I'd rather state that in a more generic way.

If you want to stick with that: appcat -> "VSHN Application Catalog"

Reply:
I have rephrased the paragraph. Does it work for you this way?

Reply:
Yes, now it makes most sense to me. Thanks.